app/models/experimentation/notifications_experiment.rb

```diff
@@ -171,6 +171,10 @@ def time(method_name, &block)
       Statsd.instance.flush # The metric is not sent to datadog until the buffer is full, hence we explicitly flush.
     end
 
+    def time(method_name, &block)
```

Comment:
you can delegate from the instance to the class

```suggestion
delegate :time, to: self
```

Reply:
Thanks!